data/scene/earth/earth.mod

```diff
@@ -78,9 +78,10 @@ return {
             -- EndTime = "2017 JAN 01 12:00:00.000",
             -- SampleInterval = 3600
             Period = 365.242,
-            Resolution = 1000
+            Resolution = 1000,
+            Tag = "defaultTrails"
```

Comment:
Let's set a good precendent on naming for tags...  How about:  `planet_trails` instead?